problems/removeVowels.js

```diff
@@ -6,8 +6,14 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
+const removeVowels = (str) => {
+    if(str === "Aa" || str === "Ee" || str === "Ii" || str === "Oo" || str === "Uu"){
```

Comment:
This checks if the whole string is "Aa" or "Ee" etc. We need to iterate through the string and make the changes. Checking only one character at a time.